src/Engine/ProtoCore/FFI/CLRDLLModule.cs

```diff
@@ -988,6 +1114,41 @@ public override FFIFunctionPointer GetFunctionPointer(string className, string n
             return null;
         }
 
+        /// <summary>
+        /// Creates CLRModuleTypes for all types in this module.
+        /// </summary>
+        /// <param name="isReflectionContext">When true the behavior for creating AST Nodes for the module types is using specific ReflectionOnly methods</param>
+        public override void ScanModule(bool isReflectionContext)
```

Comment:
so today, this is not called from any existing code right?

Reply:
right